src/posix/platform/Makefile.am

```diff
@@ -51,6 +51,7 @@ libopenthread_posix_a_SOURCES             = \
     hdlc_interface.cpp                      \
     infra_if.cpp                            \
     logging.cpp                             \
+    mainloop.cpp                            \
```

Comment:
Run `script/update-makefiles.py` to add it for gn build file?

Reply:
gn doesn't use posix platform.